core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2690,6 +2690,46 @@ public void updateMount(AlluxioURI alluxioPath, MountContext context)
     }
   }
 
+  /**
+   * Mounts an under storage path to an Alluxio path.
+   * <p>
+   * Links an under storage path to an Alluxio path,
+   * where files and folders created in Alluxio space
+   * under the path will be backed by a corresponding
+   * file or folder in the under storage path.
+   * <p>
+   * Increments {@link DefaultFileSystemMaster.Metrics#MOUNT_OPS},
+   * which keeps track of the number of mount operations made. Uses
+   * {@link LockingScheme} to determine that the path should be locked
+   * with the lock pattern {@link LockPattern#WRITE_EDGE}.
+   * <p>
+   * Attempts to instantiate {@link RpcContext} through {@link DefaultFileSystemMaster#createRpcContext()}
```

Comment:
I think this is too detailed on the implementation of the method. Comments about implementation should go within the code itself.

The method documentation should include the behavior or contract of the method, but not necessarily how it achieves that behavior.